app/controllers/account_activations_controller.rb

```diff
@@ -0,0 +1,16 @@
+class AccountActivationsController < ApplicationController
+  def edit
+    user = User.find_by(email: params[:email])
+    if user && !user.activated? && user.authenticated?(:activation, params[:id])
```

Comment:
cứ chỗ nào mà có đoạn code điều kiện đi cùng 1 đoạn tính toán thì thêm dòng trống ở giữa 2 đoạn em nhé